apps/web/src/server/applications/create/applicant/applications-create-applicant.controller.js

```diff
@@ -68,9 +68,9 @@ export async function updateApplicationsCreateApplicantTypes({ path, session, bo
  * @type {import('@pins/express').RenderHandler<ApplicationsCreateApplicantOrganisationNameProps, {}, {}, {}, {}>}
  */
 export async function viewApplicationsCreateApplicantOrganisationName(req, response) {
-	const { applicationId } = response.locals;
+	const { applicationId, applicantId } = response.locals;
 
-	const applicant = await getNthApplicant(applicationId, 0);
```

Comment:
are we still using somewhere the getNthApplicant function? if not you can remove it

Reply:
@DavidJWhiteDev ?

Reply:
yeah, I only left it in in case we needed it in the future!

Reply:
nah, you can remove it